README.md

```diff
@@ -77,6 +77,53 @@ DESTDIR="$(pwd)/Lite XL.app" meson install --skip-subprojects -C build
 Please note that the package is relocatable to any prefix and the option prefix
 affects only the place where the application is actually installed.
 
+## Installing Prebuilt
+
+Head over to [releases](https://github.com/lite-xl/lite-xl/releases) and download the version for your operating system.
+
+### Ubuntu
+
+Unzip the file and `cd` into the `lite-xl` directory:
+
+```
+tar -xf <file>
+cd lite-xl
+```
+
+Copy files over into appropriate directories:
+
+```
+mkdir -p ~/.local/bin && cp bin/lite-xl ~/.local/bin
+cp -r share ~/.local
+```
+
+If `~/.local/bin` is not in PATH:
+
+```
+echo -e 'export PATH=$PATH:~/.local/bin' >> ~/.bashrc
```

Comment:
I imagine using `$HOME` is better than `~` here

Reply:
`.bashrc`? Not everyone uses bash. And Ideally PATH should be set in .profile, and for bash at .bash_profile. 

(Bash being bash, I hardlink .bash_profile to .profile)

There is no point of modifying shell's init file to add a directory to PATH, it should be done by the user.

Reply:
> I imagine using $HOME is better than ~ here

Why?

Reply:
no particular reason. I'm just afraid that the tilde don't get expanded properly.